src/panels/Scene/SceneGraphNode/SceneGraphNodeMetaInfo.tsx

```diff
@@ -47,49 +49,61 @@ export function SceneGraphNodeMetaInfo({ uri }: Props) {
       [
         'Identifier:',
```

Comment:
Now that the TD items include text, these should probably be text too. But please make the text small to respect screenspace 

```suggestion
        <Text size={'sm'}>Identifier:</Text>,
```

And add the `size={'sm'}` part to all the other texts in here as well